packages/card/src/ui-card.component.vue

```diff
@@ -0,0 +1,86 @@
+<template>
+  <div :class="classes" :style="style">
+    <div v-if="header" :class="theme.cssClass.header">{{ header }}</div>
+    <img
+      v-if="!imgBottom && imgSrc"
+      :src="imgSrc"
+      :class="theme.cssClass.img"
+      :alt="imgAlt"
+    />
+    <div :class="bodyClasses">
+      <h5 v-if="title" :class="theme.cssClass.title">{{ title }}</h5>
+      <h6 v-if="subtitle" :class="theme.cssClass.subtitle">{{ subtitle }}</h6>
+      <slot></slot>
+    </div>
+    <img
+      v-if="imgBottom && imgSrc"
+      :src="imgSrc"
+      :class="theme.cssClass.imgBottom"
+      :alt="imgAlt"
+    />
+    <div v-if="footer" :class="theme.cssClass.footer">
+      {{ footer }}
+    </div>
+  </div>
+</template>
+
+<script setup lang="ts">
+import { computed, defineProps, useAttrs, toRefs } from 'vue'
```

Comment:
`defineProps` and `defineEmits` are compiler macros only usable inside `<script setup>`. They do not need to be imported and are compiled away when <script setup> is processed

Reply:
I'll remove it